getting-started/display.rst

```diff
@@ -132,8 +130,7 @@ Knowing a specific data ID, let's get the dataset with the Butler client's ``get
 .. code-block:: python
 
    import os
-   collection = f"u/{os.environ['USER']}/single_frame"
-   calexp = butler.get('calexp', visit=23718, detector=41, collections=collection, instrument='HSC')
+   calexp = butler.get('calexp', visit=23718, detector=41, instrument='HSC')
```

Comment:
FYI, one can also pass `instrument='HSC'` when constructing the `Butler` and then never have to pass it again, just as with collections (also works with `skymap`).